person.rb

```diff
@@ -53,4 +54,8 @@ def correct_name
     original_name = @nameable.correct_name
     original_name.length > 10 ? original_name[0..9] : original_name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] @bill7pearl great work implementing the changes requested, however, you have added the add rentals in the wrong class (TrimmerDecorator) instead of the requested person class, also you need to pass self, book, and date as parameters when creating the rental.

Reply:
- [ ] [optional] @bill7pearl in order to make your code readable and neet L would advice you to create a file for each class .

Reply:
done